kubejs/server_scripts/enigmatica/kubejs/expert/recipetypes/minecraft/shaped.js

```diff
@@ -100,6 +100,18 @@ onEvent('recipes', (event) => {
                 A: 'kubejs:red_nether_brick'
             },
             id: 'minecraft:red_nether_bricks'
+        },
+        {
+            output: 'minecraft:enchanting_table',
+            pattern: [' A ', 'BCB', 'DED'],
+            key: {
+                A: 'ars_nouveau:novice_spell_book',
+                B: 'atum:isis_godshard',
+                C: 'eidolon:worktable',
+                D: 'minecraft:crying_obsidian',
+                E: 'quark:diamond_heart'
```

Comment:
How hard is this to obtain?

Reply:
Fairly common drop from stonelings. I wouldn't say hard, but potentially annoying.